app/resources/start.html.template

```diff
@@ -19,14 +19,14 @@
     <div class="row">
       <div class="action-outer start">
         <a href="{{seek_url}}" tabindex="1">
-          <div class="action-inner">
+          <div class="action-inner" onclick="location.href={{seek_url}};" tabindex="1" style="cursor:pointer;">
```

Comment:
I guess it's better to do this for "action-outer" DIV instead? "action-outer" corresponds to the whole button.

Also, add a comment explaining that this is to make the whole button clickable.

I guess you need single quotes around the URL. Also, put whitespaces around `=` in JS code: `location.href={{seek_url}};` -> `location.href =
 '{{seek_url}}';`

I guess you don't need tabindex attribute here. It' sufficient if the A tag has tabindex.

Put a whitespace after `:`: `cursor:pointer` -> `cursor: pointer`